Rubberduck.Core/UnitTesting/TestEngine.cs

```diff
@@ -107,51 +107,44 @@ private void RunInternal(IEnumerable<TestMethod> tests)
                                 && test.Declaration.QualifiedName.QualifiedModuleName.ComponentName == capturedModule.Key.ComponentName);
 
                 var fakes = _fakesFactory.Create();
-                try
+                Run(module.Key.FindModuleInitializeMethods(_state));
```

Comment:
Shouldn't all `Run` methods be inside a `try` block? Or is it not needed here if we are simply updating the ignored tests' result?

Reply:
Actually, I also realize this is inside the `RunInteral` and we are calling `Run`. I do not think that is right and I should have caught that earlier. I think all `Run` methods within the `RunInternal` needs to be updated to avoid going through the `UiDispatcher.InvokeAsync` again?

Reply:
The code is rather confusing, but the calls to `Run` inside `RunInternal` use the overload taking `IEnumerable<Declaration>`, which you find below `RunInternal`. They are not going through the `UiDispatcher`.